### PR TITLE
Add render graph rewrite to What's Next

### PR DESCRIPTION
**Pull request by ecoskey**

### CODE REVIEW

File: content/news/draft-bevy-0.14/index.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ Peering deep into the mists of time (predictions are *extra* hard when your team
...
 - **A unified view on dev tools:** In 0.14, we've added a stub `bevy_dev_tools` crate: a place for tools and overlays that speed up game development such as performance monitors, fly cameras, or in-game commands to spawn game objects. We're working on adding more tools, and creating a [dev tool abstraction](https://github.com/bevyengine/rfcs/pull/77). This will give us a unified way to enable/disable, customise and group this grab bag of tools into toolboxes to create something like Quake console or VSCode Command Palette with tools from around the ecosystem.
+- **A modular, maintainable render graph:** Bevy's existing rendering architecture is already quite good at providing reusable renderer features such as RenderPhases, batching, and draw commands. However, the render graph itself is one area that is greatly lacking. While the exact design hasn't been finalized (and feedback is greatly encouraged!!!), we've been actively working to [redesign the render graph](https://github.com/bevyengine/bevy/pull/13397) in order to build up to a larger refactor of the renderer towards modularity and ease of use.
```
Comment (alice-i-cecile) [inline content/news/draft-bevy-0.14/index.md:39]:
```suggestion
- **A modular, maintainable render graph:** Bevy's existing rendering architecture is already quite good at providing reusable renderer features like `RenderPhases`, batching, and draw commands. However, the render graph itself is one area that is greatly lacking. While the exact design hasn't been finalized (and feedback is very welcome!), we've been actively working to [redesign the render graph](https://github.com/bevyengine/bevy/pull/13397) in order to build up to a larger refactor of the renderer towards modularity and ease of use.
```

Comment (alice-i-cecile) [reply content/news/draft-bevy-0.14/index.md:39]: Small edits for readability.

Comment (alice-i-cecile) [inline content/news/draft-bevy-0.14/index.md:39]: I'd like to hear more about why the render graph is currently bad in here: another half-sentence or so is all that's needed.

Comment (ecoskey) [reply content/news/draft-bevy-0.14/index.md]: done!

**Review by alice-i-cecile — APPROVED** [review]

**Pull request merged**